### ISSUE TITLE
bug(metrics): bad 

### ISSUE THREAD

**Issue by fuyufjh** [risingwavelabs/risingwave#12293]
### Describe the bug

Invalid data reported: `lru_evicted_watermark_time_diff_ms`

<img width="950" alt="image" src="https://github.com/risingwavelabs/risingwave/assets/10192522/7fffd9ea-eaca-4034-930b-a9f4a8745ee5">

**Comment by yuhao-su** [comment]
It's likely the node restarted at those data peaks.

**Comment by fuyufjh** [comment]
Yeah. The value is so huge that makes the normal values unable to be seen. I'm considering we should exclude the in some ways

**Comment by yuhao-su** [comment]
> The value is so huge that makes the normal values unable to be seen

Yeah I failed to consider this. Will fix this.
